sky/data/data_transfer.py

```diff
@@ -88,8 +80,31 @@ def s3_to_gcs(s3_bucket_name: str, gs_bucket_name: str) -> None:
         }
     }
 
-    result = storagetransfer.transferJobs().create(body=transfer_job).execute()
-    logger.info(f'AWS -> GCS Transfer Job: {json.dumps(result, indent=4)}')
+    response = storagetransfer.transferJobs().create(
+        body=transfer_job).execute()
+    operation = storagetransfer.transferJobs().run(jobName=response['name'],
+                                                   body={
+                                                       'projectId': project_id
+                                                   }).execute()
+
+    logger.info(f'{colorama.Fore.GREEN}Transfer job scheduled: '
+                f'{colorama.Style.RESET_ALL}'
+                f's3://{s3_bucket_name} -> gs://{gs_bucket_name} ')
+    logger.debug(json.dumps(operation, indent=4))
+    logger.info('Waiting for the transfer to finish')
+    start = time.time()
+    with backend_utils.safe_console_status('Transferring'):
+        for _ in range(MAX_POLLS):
+            result = (storagetransfer.transferOperations().get(
+                name=operation['name']).execute())
+            if 'error' in result:
+                with ux_utils.print_exception_no_traceback():
+                    raise RuntimeError(result['error'])
+
+            if 'done' in result and result['done']:
+                break
+            time.sleep(POLL_INTERVAL)
+    logger.info(f'Transfer finished in {time.time() - start:.2f} seconds')
```

Comment:
nit: What is the typical duration of these jobs? If its greater than 60 seconds, should we print minutes here instead of seconds?

Reply:
Good point! Changed it to minutes. Thanks!